web-app/app.js

```diff
@@ -1,3 +1,11 @@
+const Sequelize = require("sequelize");
+
+const sequelize = new Sequelize({
+  dialect: "sqlite",
+  storage: '../../../database/plant.db'
```

Comment:
Using a relative path here is a good idea, but keep the data inside of the project folder. I am guessing that you did this because you didn't want to add the database to the Git repo by accident (which is correct), but the better way to do this is just add a line to `.gitignore` so that it doesn't get added. In addition, I would force the empty directory to exist in the repo by adding a `.keep` file. So here is what I would do:

* Change this to `../database/plant.db`
* Add `database/*` to `.gitignore`
* Add `database/.keep` to the git repo